stonesoup/updater/particle.py

```diff
@@ -30,8 +31,14 @@ class ParticleUpdater(Updater):
     """
 
     resampler: Resampler = Property(default=None, doc='Resampler to prevent particle degeneracy')
-    regulariser: Regulariser = Property(default=None, doc="Regulariser to prevent particle "
-                                                          "impoverishment")
+    regulariser: Regulariser = Property(
+        default=None,
+        doc='Regulariser to prevent particle impoverishment. The regulariser '
+            'is normally used after resampling. If a class:`~.Resampler` is defined,'
+            ' then regularisation will only take place if the particles have been '
```

Comment:
minor documentation change

```suggestion
            'is normally used after resampling. If a class:`~.Resampler` is defined, '
            'then regularisation will only take place if the particles have been '
```